http_service/bugbug_http/worker.py

```diff
@@ -7,11 +7,15 @@
 import os
 import sys
 
+import sentry_sdk
 from redis import Redis
 from rq import Connection, Worker
 
 import bugbug_http.boot
 
+if os.environ.get("SENTRY_DSN"):
+    sentry_sdk.init(os.environ.get("SENTRY_DSN"))
```

Comment:
In code-review bot, we set `release=VERSION`, so we can associate errors to versions, it would be beneficial here too